release-team/role-handbooks/patch-release-manager/README.md

```diff
@@ -1,41 +1,98 @@
 # Patch Release Manager Playbook
 
-This is a playbook intended to guide new patch release managers.
-It consists of opinions and recommendations from former patch release managers.
-
-Note that patch release managers are ultimately responsible for carrying out
-their [duties](https://github.com/kubernetes/sig-release#patch-release-manager)
-in whatever manner they deem best for the project.
-The playbook is more what you call "guidelines" than actual rules.
+As the name implies, Patch Release Managers are responsible for
+managing patches against Kubernetes release branches and making the
+1.X.Y patch releases during the support period after each 1.X minor
+release.  Kubernetes 1.X releases receive approximately 9 months
+of support in terms of patches for bugfixes and ongoing CI insuring
+the branch's health and the ability to update from 1.X.Y to 1.(X+1).Y,
+for the newest Y on each of those two branches.
+
+Specific duties of the Patch Release Manager include:
+
+- Ensuring the release branch (e.g. `release-1.13`) remains in a
+  healthy state as measured by the version specific release-blockin
+  CI (e.g. [testgrid sig-release-1.13-blocking
```

Comment:
Can we explicitly link all the dashboards to watch here? "E.g." implies it's not a complete list, therefore there's still some folk knowledge left.

Reply:
I'd like to avoid having to edit this document every three months to remove 1.(X-3) and add 1.X to lists.  At the same time I want to give at least a working url, so was using 1.13 as the example version where the link is version specific.

For this url, I've tried to clarify the english to show that is the one board to look at for 1.13 branch health.

Is there a better way?

Reply:
Does the updated text look any better now?

Reply:
Example as in "1.13 is used as example here" makes more sense, thanks for clarifying this. Regarding the link, I suppose it could also be https://k8s-testgrid.appspot.com/sig-release, but it seems to default to sig-release-master-blocking for now, so it's probably not much better.